vendor/bat-native-ledger/src/bat/ledger/internal/state/state.cc

```diff
@@ -386,5 +386,39 @@ bool State::GetAnonTransferChecked() {
   return ledger_->ledger_client()->GetBooleanState(kAnonTransferChecked);
 }
 
+absl::optional<std::string> State::GetEncryptedString(const std::string& key) {
+  std::string value = ledger_->ledger_client()->GetStringState(key);
+  if (value.empty())
+    return "";
```

Comment:
Should this be `return {};`?

Reply:
Left a comment in the code - we want to return an empty string here so that callers can distinguish between a successful read of an empty pref and an unsuccessful read.

Reply:
Ah got it, I missed that.